index.js

```diff
@@ -0,0 +1,30 @@
+const formElement = document.querySelector('.editForm');
+const  nameInput = document.querySelector('#nameInput');
+const  jobInput = document.querySelector('#jobInput');
```

Comment:
id положи в константу
```javascript
const IDS = {
  NAME_INPUT: 'nameInput',
  JOB_INPUT: 'jobInput',
};
```
```suggestion
const  nameInput = document.querySelector(`#${IDS.NAME_INPUT}`);
const  jobInput = document.querySelector(`#${IDS.JOB_INPUT}`);
```